tests/test_phptravels.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+import time
+
+class ByLocators:
+    TAG_NAME = By.TAG_NAME, "tag name from the project to find element by"
+    ID = By.ID, "Id from the project to find element by"
+    XPATH = By.XPATH, "Xpath from the project to find element by"
+    CSS_SELECTOR = By.CSS_SELECTOR, "CSS selector from the project to find element by"
+    NAME = By.NAME, "Element name from the project to find element by"
+    CLASS_NAME = By.CLASS_NAME, "Class name from the project to find element by"
+    LINK_TEXT = By.LINK_TEXT, 'Link search, to search hyperlinks by the full hyperlink text '
+    PARTIAL_LINK_TEXT = By.PARTIAL_LINK_TEXT, 'Link search, tto search hyperlinks by the partial hyperlink tex'
+
+
+class TestLoginWithCorrectCreds(unittest.TestCase):
+    def setUp(self):
+        self.driver = webdriver.Chrome()
+        # self.driver = webdriver.Firefox()
+
+
+    def tearDown(self):
+        self.driver.quit()
+
+
+    def test_login(self):
+        self.driver.get('https://phptravels.com/demo/')
```

Comment:
'https://phptravels.com/demo/' better to move in class attribute in separate value

Reply:
ok, it is done